.circleci/config.yml

```diff
@@ -1,64 +1,12 @@
-# Python CircleCI 2.0 configuration file
+# Python CircleCI 2.1 configuration file
 #
 # Check https://circleci.com/docs/2.0/language-python/ for more details
```

Comment:
As we change CI to 2.1 , should we change this url ?

Reply:
Seems like 2.0 documentation is the latest